src/persist-client/Cargo.toml

```diff
@@ -39,8 +39,11 @@ futures-util = "0.3"
 mz-ore = { path = "../ore" }
 mz-persist-types = { path = "../persist-types" }
 mz-persist = { path = "../persist" }
+mz-repr = { path = "../repr" }
```

Comment:
I would greatly like to avoid depending on mz-repr in persist-client. let's talk!

Reply:
I was indeed able to eliminate `ProtoShardId` entirely! (turns out that code you write in your seventh hour of work of the day isn't that great, who would have thought)